### DISCUSSION TITLE
Docker Image

### DISCUSSION THREAD

**Discussion by ralphyboy84** [webdriverio/webdriverio#9194] [General]
Is there a docker image with webdriver already built into it?

**Comment by KiThoSivaPrakash** [comment]
you can refer this boilerplate project for running WebdriverIO inside docker container https://github.com/praveendvd/webdriverIODockerBoilerplate

**Comment by berekmerilorand** [comment]
Maybe [this](https://hub.docker.com/r/huli/webdriverio) can help.
